app/src/main/java/net/ontariotechu/food_e/ui/BrowseFragment.java

```diff
@@ -111,7 +97,13 @@ private void onMealChipChanged(ChipGroup group, List<Integer> checkedIds) {
             Chip chip = group.findViewById(id);
             selectedFilters.get("mealType").add(chip.getText().toString());
         }
-        dataPasser.onFilterChanged(selectedFilters);
+        recipeService.getRecipesBackground(etSearch.getText().toString(), selectedFilters, (result) -> {
```

Comment:
Should this just call on Recipe Result?  Why does this need to be different than the cuisine chip?

Reply:
You're right I missed this when I refactored it